test/integration/index_management.test.ts

```diff
@@ -770,20 +770,19 @@ describe('Indexes', function () {
 
           expect(events).to.be.an('array').with.lengthOf(1);
           expect(events[0]).nested.property('command.commitQuorum').to.equal(0);
-          await collection.drop(err => {
-            expect(err).to.not.exist;
-          });
+          await collection.drop();
         }
       };
     }
     it(
       'should run command with commitQuorum if specified on db.createIndex',
-      commitQuorumTest((db, collection) =>
-        db.createIndex(collection.collectionName, 'a', {
-          // @ts-expect-error revaluate this?
-          writeConcern: { w: 'majority' },
-          commitQuorum: 0
-        })
+      commitQuorumTest(
+        async (db, collection) =>
```

Comment:
nothing's changed here or above (sort of a correctness fix above) but now this prints a better stack trace when I was debugging the issue.